packages/manager/modules/overthebox/src/overthebox/autoconfigure/dns/template.html

```diff
@@ -0,0 +1,249 @@
+<div>
+    <oui-message
+        data-ng-if="$ctrl.successMessage"
+        type="success"
+        aria-close-button-label="{{:: 'overTheBox_autoconfigure_close' | translate}}"
+        on-dismiss="$ctrl.onDismiss()"
+        dismissable
+    >
+        <span data-ng-bind="$ctrl.successMessage"></span>
+    </oui-message>
+    <oui-message
+        data-ng-if="$ctrl.errorMessage"
+        type="error"
+        aria-close-button-label="{{:: 'overTheBox_autoconfigure_close' | translate}}"
+        on-dismiss="$ctrl.onDismiss()"
```

Comment:
I don't see any `onDismiss` method into the controller, is there any?

Reply:
I don't see it either ^^